### ISSUE TITLE
[FR] Align file icons and colors

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/android#4483]
**Problem**
Icons and Icon colors are quite different between web and mobile clients. This is particularly noticeable when I e.g. do presentations, see screenshot.

![screenshot_003187](https://github.com/user-attachments/assets/18053ee2-d336-4d81-9097-f3a905f7c331)

**Proposed solution**

Align icon style and colors between web and mobile clients.

![android](https://github.com/user-attachments/assets/27ffe347-01cf-4f76-8e0c-fbc40627e9fe)

@JuancaG05 @jesmrec Whats you opinion - do you consider the proposal an improvement?

**Comment by jesmrec** [comment]
yes, that's an improvement. We just need those icons from somewhere.

**Comment by tbsbdr** [comment]
We could proceed like that:
1. replace current icons with the proposed one (same as web)
2. discuss centralised icon set (not sure if this approach will be a benefit, as clients will always need some set of local icons - tbd)

The proposed icons are taken from web, `resource-type-*.svg`
- [https://github.com/owncloud/web/blob/master/packages/design-system/src/assets/icons/resource-type-pdf-fill.svg](https://github.com/owncloud/web/blob/master/packages/design-system/src/assets/icons/resource-type-pdf-fill.svg
) I've extracted the colors web via browser inspect eg from: https://ocis.ocis-web.master.owncloud.works/ 

mabye @kulmann and team can provide more structured guidance 🙏

**Comment by Kishubhai** [comment]
Issue gets solved?

**Comment by tbsbdr** [comment]
No

**Comment by tbsbdr** [comment]
I can not tell, @DeepDiver1975 might know or might know who can decide. (Im no longer in charge)

**Comment by joragua** [comment]
This issue has been assigned to the Android Team and it's in progress! 🚀 @Kishubhai if you want to contribute, take a look to [issues with `Contributions are welcome` label](https://github.com/owncloud/android/issues?q=is%3Aissue+is%3Aopen+label%3A%22Contributions+are+welcome%22). Feel free to comment if you have some questions about those issues 😄

**Comment by JuancaG05** [comment]
Seems the provided SVG are not valid since they are rectangular, and we (and in general, mobile apps) need squared images for these icons. The vectors could be modified before introducing them into the app, making them squared, but that's not the aim here, we should keep the same resources used in other clients. This won't be implemented FTM.
